slices/grep.yaml

```diff
@@ -0,0 +1,12 @@
+package: grep
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - libpcre3_libs
+    contents:
+      /bin/egrep:
```

Comment:
`egrep`, `fgrep`, and `rgrep` are all shell scripts that depend on `#!/bin/sh` being installed - this needs to be a manually created symlink as per this [comment](https://github.com/canonical/chisel-releases/blob/47a2d67283e10f1a7494d5c8c023e073e0d8147a/slices/bash.yaml#L17)

Should these be excluded since they won't work as-is?